### PR TITLE
removing any spacing around photo detail #148

### PR DESCRIPTION
**Pull request by Szeket**
And breaking titles so they display nicely even if they are long.

### CODE REVIEW

File: src/components/results/detail/ImageDetail.vue [hunk 1]
```diff
@@ -46,11 +46,12 @@
...
                       offset-md="1"
                     >
-                      <div
-                        :class="{ blurExplicit: blurExplicit(file)}"
+                      <img
+                        :src="resourceURL"
                         class="image-wrapper"
-                        :style="{'background-image': `url(${resourceURL})`}"
-                      />
+                        :class='{ blurExplicit: blurExplicit(file)}'
```
Comment (femans) [inline src/components/results/detail/ImageDetail.vue:52]: here there should be double quotes

File: src/components/results/detail/helpers/MediaHeader.vue [hunk 2]
```diff
@@ -43,9 +44,7 @@ import DetailMixin from '@/components/results/detail/mixins/DetailMixin';
 import CopyHashButton from '@/components/helpers/CopyHashButton';
 
 export default {
-  mixins: [
-    DetailMixin,
-  ],
+  mixins: [DetailMixin],
```
Comment (femans) [inline src/components/results/detail/helpers/MediaHeader.vue:47]: This can be left like this for now, but is an unnecessary change

File: src/components/results/detail/helpers/MediaHeader.vue [hunk 3]
```diff
@@ -43,7 +44,7 @@ import DetailMixin from '@/components/results/detail/mixins/DetailMixin';
 import CopyHashButton from '@/components/helpers/CopyHashButton';
 
 export default {
-  mixins: [
+mixins: [
```
Comment (femans) [inline src/components/results/detail/helpers/MediaHeader.vue:47]: There is a space error here

File: src/components/results/detail/ImageDetail.vue [hunk 4]
```diff
@@ -46,11 +46,12 @@
...
-                      <div
-                        :class="{ blurExplicit: blurExplicit(file)}"
+                      <img
+                        :src="resourceURL"
                         class="image-wrapper"
-                        :style="{'background-image': `url(${resourceURL})`}"
-                      />
+                        :class="{ blurExplicit: blurExplicit(file)}"
+                      >
+
```
Comment (femans) [inline src/components/results/detail/ImageDetail.vue:54]: would you mind removing this empty line still? Then it's perfect.

Comment (Szeket) [reply src/components/results/detail/ImageDetail.vue]: sure!
